src/commands/release/import.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license
+ * Copyright 2016-2024 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Flags, Args } from '@oclif/core';
+import Command from '../../command';
+import * as cf from '../../utils/common-flags';
+import { getBalenaSdk, stripIndent } from '../../utils/lazy';
+import { apply } from '@balena/release-bundle';
+import { createReadStream } from 'fs';
+import { ExpectedError } from '../../errors';
+
+export default class ReleaseImportCmd extends Command {
+	public static description = stripIndent`
+		Imports a release from a file to an application or fleet.
+
+		To save a release into a file, use 'balena release export'.
+
+		Use the --override-version option to specify the version 
+		of the imported release, overriding the one saved in the file.
+`;
+	public static examples = [
+		'$ balena release import ../path/to/release.tar -f 1234567',
+		'$ balena release import ../path/to/release.tar -f myFleet',
+		'$ balena release import ../path/to/release.tar -f myOrg/myFleet',
+		'$ balena release import ../path/to/release.tar -f myOrg/myFleet -V 1.2.3',
+	];
+
+	public static usage = 'release import <file>';
+
+	public static flags = {
+		fleet: { ...cf.fleet, exclusive: ['device'] },
+		'override-version': Flags.string({
+			description:
+				'Imports this release with the specified version overriding the version in the file.',
+			char: 'V',
+			required: false,
+		}),
+		help: cf.help,
+	};
+
+	public static args = {
+		bundle: Args.string({
+			required: true,
+			description: 'path to a file, e.g. "release.tar"',
+		}),
+	};
+
+	public static authenticated = true;
+
+	public async run() {
+		const { args: params, flags: options } = await this.parse(ReleaseImportCmd);
+
+		const balena = getBalenaSdk();
+
+		const bundle = createReadStream(params.bundle);
+
+		try {
+			if (
+				typeof options.fleet !== 'number' &&
+				typeof options.fleet !== 'string'
+			) {
+				throw new ExpectedError('Fleet must be a number or slug.');
+			}
```

Comment:
> I believe we have been trying to avoid allowing id to be used for identifying fleets, I would remove this example

As I noted here ^

This leaves us with just string as a type anyway. We can remove this if altogether.

Then after the 
```
const application = await balena.models.application.get(options.fleet, {
				$select: ['id'],
			});
			```
			We can add this if you really want to check this:
```
// Check app exists
		if (!application) {
			throw new ExpectedError(`Error: fleet ${params.fleet} not found.`);
		}
		```

Reply:
Thanks for the suggestion.  I included it.